src/Lavary/Menu/Builder.php

```diff
@@ -524,7 +532,7 @@ public function render($type = 'ul', $parent = null)
 			}
 					
 			if( $item->hasChildren() ) {
-				$items .= "<{$type}>";
+				$items .= ($this->style == 'adminLTE') ?  "<{$type} class='treeview-menu'>" : "<{$type}>";
```

Comment:
Can you generalize this? Such as `$items .= "<".$type.(empty($type_class)?'':" class='{$type_class}'").">"` or something other than a specific treeview-menu class?